migrator/migrations/m_169_to_m_170_collections_sac_resource_migration/migration.go

```diff
@@ -0,0 +1,68 @@
+package m169tom170
+
+import (
+	"context"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/migrator/migrations"
+	permissionSetPostgresStore "github.com/stackrox/rox/migrator/migrations/m_169_to_m_170_collections_sac_resource_migration/permissionsetpostgresstore"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/sac"
+)
+
+const (
+	batchSize = 500
+
+	workflowAdminResource       = "WorkflowAdministration"
+	reportConfigurationResource = "ReportConfiguration"
+
+	startSeqNum = 169
```

Comment:
```suggestion
	startSeqNum = migrations.CurrentDBVersionSeqNum()
```

Reply:
That function does not exist, and that's not the form of the previous (168 to 169) migration

Reply:
It exists in the `seq_num`, but the real reason we cannot follow the above suggestion is because sequence number changes with each migration.